datahub/interaction/admin_csv_import/row_form.py

```
@@ -296,6 +306,32 @@ def _populate_adviser(self, data, adviser_field, team_field):
         except ValidationError as exc:
             self.add_error(adviser_field, exc)
 
+    def _populate_service_answers(self, data):
+        """Transform service_answer into service_answers dictionary."""
+        service = data.get('service')
+        if service and service.interaction_questions.count() > 0:
```

Shouldn't we also return an error if a `service_answer` is provided but the service has no service answers?

---

Good idea - updated. Thanks!